api/testdata/complete-scenarios/test5.json

```
@@ -6853,8 +6855,9 @@
                       "first": "",
                       "firstInitialOnly": false,
                       "middle": "",
+                      "hideMiddleName": true,
                       "middleInitialOnly": false,
-                      "noMiddleName": false,
+                      "noMiddleName": true,
```

To remain consistent with others:
```suggestion
                      "noMiddleName": false,
```